assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -45,7 +47,30 @@ export default {
         "header-chat-link",
         "chat:rerender-header"
       );
+
       api.addToHeaderIcons("header-chat-link");
+
+      api.decorateCookedElement((elem) => {
```

Comment:
Btw, not sure how / when this api method gets invoked, does it show up well for chat messages?

The doc says: `Used for decorating the `cooked` content of a post after it is rendered`

Reply:
@nattsw I've added a screenshot to the OP, this BBCode is used within a post because we are quoting chat messages there. This method gets invoked for each post in the stream.

Reply:
Got it. You even said `quoting chat messages into regular posts` I should read better.